chpl/chpl-service/src/main/java/gov/healthit/chpl/app/ParseActivities.java

```diff
@@ -420,7 +504,25 @@ else if(dto.getYear().equals("2015")){
 	public void setEmailProperties(){
 		 email.setEmailTo(props.getProperty("summaryEmail").toString().split(";"));
 		 email.setEmailSubject("CHPL - Weekly Summary Statistics Report");
-		 email.setEmailMessage(summaryOutputTable.getTable());
+		 Calendar calendarCounter = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+		 String htmlPreText = "<pre><br>";
```

Comment:
Removing the pre tag improves the appearance of the email somewhat because it removes some unnecessary spacing.